apps/test/unit/code-studio/pd/foorm/FoormEditorTest.js

```diff
@@ -32,10 +32,19 @@ describe('FoormEditor', () => {
 
     store = getStore();
 
+    const headerTitle = React.createElement('h1', null, 'A title');
+
     defaultProps = {
       populateCodeMirror: () => {},
       resetCodeMirror: () => {},
-      formCategories: ['surveys/pd', 'surveys/teacher']
+      categories: ['surveys/pd', 'surveys/teacher'],
```

Comment:
These tests are probably now more accurately described as "testing the form-specific implementation of `FoormEditor`"